Assets/AppCenter/Editor/AppCenterPostBuild.cs

```diff
@@ -31,6 +31,10 @@ public class AppCenterPostBuild : IPostprocessBuild
     private const string CapabilityElement = "Capability";
     private const string CapabilityNameAttribute = "Name";
     private const string CapabilityNameAttributeValue = "internetClient";
+    private const string App_NET_D3D = "App.cs";
+    private const string App_NET_XAML = "App.xaml.cs";
+    private const string App_IL2CPP_XAML = "App.xaml.cpp";
+    private const string App_IL2CPP_D3D = "App.cpp";
```

Comment:
Align the naming style.

Reply:
Thanks for the review, unfortunately we've already merged the changes and will fix these files names in the next PR